apps/oracles/cex-price-feeds/spin.toml

```diff
@@ -4431,21 +4431,7 @@ data = '{"pair":{"base":"USDC","quote":"USD"},"decimals":8,"category":"Crypto","
 [component.cex-price-feeds]
 source = "../target/wasm32-wasip1/release/cex_price_feeds.wasm"
 allowed_outbound_hosts = [
-  "https://api.kraken.com",
-  "https://api.bybit.com",
-  "https://api.coinbase.com",
-  "https://api.exchange.coinbase.com",
-  "https://api1.binance.com",
-  "https://api.kucoin.com",
-  "https://api.mexc.com",
-  "https://api.crypto.com",
-  "https://api.binance.us",
-  "https://api.gemini.com",
-  "https://api-pub.bitfinex.com",
-  "https://api.upbit.com",
-  "https://api.bitget.com",
-  "https://api.gateio.ws",
-  "https://www.okx.com",
+  "http://127.0.0.1:3000",
```

Comment:
Are we going to reintroducte this list of allowed outbound hosts in the new proxy somehow? We can start by *hardcoding* it in `Rust` directly (for now), leaving the possibility to abstract it away in a configuration file somewhere, similar to this one.